### DISCUSSION TITLE
Login error: "This connection is not secure. Logins entered here could be compromised"

### DISCUSSION THREAD

**Discussion by dlegranddk** [firefly-iii/firefly-iii#8481] [General]
Hi,

Thanks to the developer for the work, I'm finishing the TEST phase and will [subscribe](https://www.patreon.com/jc5) as I "go live".

I know this question has been asked many times such as [here](https://github.com/firefly-iii/firefly-iii/issues/2162) or [here](https://github.com/orgs/firefly-iii/discussions/5118) and I don't expect the developer to be able to help me but maybe someone is running the software on Synology/Docker like me and solved it. And that's the last big issue I have.

Situation:
* Synology NAS : as I can't figure out yet how to get treafik or others to get all the inbound traffic, I still uses the "Login portal" which is a UI in front of nginx with the following config found [in the doc](https://docs.firefly-iii.org/references/faq/install/) and one of the post
![Proxy 1](https://github.com/firefly-iii/firefly-iii/assets/104696958/8f4d6282-c63b-4e07-a23e-4cb055a6dc12)
![Proxy 2](https://github.com/firefly-iii/firefly-iii/assets/104696958/ed274443-340e-44e7-bad2-735bf8ea4ecc)
![Proxy 3](https://github.com/firefly-iii/firefly-iii/assets/104696958/d7ee3606-715b-4e0b-8f49-e4bc76de2380)

I have tried to figure out where Synology store the actual nginx config and to update it through the console but without more success

* Docker compose using of course:
* TRUSTED_PROXIES=**
* APP_URL=http://localhost (as the .env tells to keep it as is) OR https://fireflyiii.mydomain.host as I wasn't sure how to interpret [this post](https://github.com/firefly-iii/firefly-iii/issues/1869#issuecomment-438624792)

And so with this config, when I try to go to https://fireflyiii.mydomain.host, I end up with "Your connexion isn't fully secure"
![Unsecured page](https://github.com/firefly-iii/firefly-iii/assets/104696958/7c3b5a56-d104-4fd6-a6b3-5ab3195edcad)

The setup Docker + "Synology nginx" works for other services I installed so far but nothing else uses Laravel and I couldn't find anything relevant on the net for that either.
If no solution can be found, I'll try to buy a Raspberry Pi to host the reverse proxy outside of the Synology.

Best regards,

**Comment by JC5** [comment]
Have you set `TRUSTED_PROXIES=*`?

**Reply by dlegranddk** [reply]
Hi James,

Thanks for answering. yes, I tried both TRUSTED_PROXIES=** and TRUSTED_PROXIES=* in the .env as I saw both options in various comments.
It’s probably more a Nginx synology issue but as I only see it with your software, I allowed myself to create a discussion to see if others have had the issue.

After my go live, I’ll also try to list the few actions/tricks I have done to automate, it might be useful for newcomers. But it’s a great product, in a couple of weeks, I have been able to import several years of correct and correlated data, I’m “lucky” to live in an almost cash-free country.

**Reply by JC5** [reply]
It seems you're using the right configuration. Weird! I'm not sure what the next step could be :( 

If you find anything, let me know!
